### PR TITLE
Picked Values from yml file to all text in UI.

### PR DESCRIPTION
**Pull request by SwarajPawarV2Solutions**
I have successfully completed the task of updating the YAML file to include values for all text elements in the UI. Specifically, I have updated the YAML file for the first page, which is the login page.

### CODE REVIEW

File: app/views/pages/home.html.erb [hunk 1]
```diff
@@ -1,9 +1,30 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title><%= t('login.title')%></title>
+    <%= stylesheet_link_tag "pages_home" %>
+   
+    
+</head>
+<body>
```
Comment (manojpm) [inline app/views/pages/home.html.erb]: What is the need to include the hml, head tags ?

**Review by shilesh — CHANGES_REQUESTED** [review]
Address PR comments

1. Use templates for specific contents through `yield` 
2. Take latest from `main` and resolve conflicts

**Comment by SwarajPawarV2Solutions** [comment]
As you mentioned in  the previous comment , I have used templates for specific contents  like header and footer through yield and also resolved Conflicts and pushed code in swaraj_yml_to_ui branch.

File: app/views/pages/home.html.erb (hunk 1 above)
Comment (SwarajPawarV2Solutions) [reply app/views/pages/home.html.erb]: Yes  sir you are right i am agreed with you that there is no need to include html, head tags. code will work without this tags also. so i have updated the code(removed tags) and pushed on github.

**Pull request merged**